### ISSUE TITLE
[css-backgrounds] background-size-007.html and background-size-010.html to be improved

### ISSUE THREAD

**Issue by TalbotG** [web-platform-tests/wpt#22205]
[background-size-007.html](http://wpt.live/css/css-backgrounds/background-size-007.html)

and

[background-size-010.html](http://wpt.live/css/css-backgrounds/background-size-010.html)
should be coded and tuned to use 

../reference/ref-nothing-below.xht

as the reference file since both tests expect/predict that "background image is not to be displayed". The overlapping technique is illogical in these 2 tests.

New code:

  `<link rel="match" href="../reference/ref-nothing-below.xht">`
(...)
```
  div
    {
      background-image: url(support/60x60-red.png);
      background-size: 0px;
      height: 100px;
      width: 100px;
    }
```
(...)
```
  <p>Test passes if there is nothing below.
  <div></div>
```

**Comment by TalbotG** [comment]
These 2 tests, as coded right now, are not reliable and not trustworthy.

**Comment by frehner** [comment]
from what I can tell, this issue was fixed in #22245 and could be closed now, correct?

**Comment by TalbotG** [comment]
@frehner 
Yes, Anthony. Closing now.
